.devcontainer/Dockerfile

```diff
@@ -20,6 +26,12 @@ RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
     python3-wheel \
     wget \
     zip \
+    usbutils \
+    hwdata \
+    gpg \
+    gnupg2 \
     && apt-get clean && rm -rf /var/lib/apt/lists/*
 
-RUN pipx install platformio==6.1.15
\ No newline at end of file
+RUN pipx install platformio==6.1.15
```

Comment:
maybe add `install --upgrade meshtastic` ? :)

Reply:
I think there's a chance that might conflict, since this is designed for dev-envs?